2018-05-Berlin/attendee-rsvp.md

```diff
@@ -0,0 +1,89 @@
+20. [Danese Cooper](https://github.com/Danese)
+21. [Thomas Watson](https://github.com/watson)
+22. [Stephen Belanger](https://github.com/Qard)
+23. [Avraam Mavridis](https://github.com/AvraamMavridis)
+24. [Addison Berry](https://github.com/add1sun)
+25. [Yang Guo](https://github.com/hashseed)
+26. [Daniel Bevenius](https://github.com/danbev)
+27. [Abraham Agiri Jr.](https://github.com/codeekage)
+28. [Gibson Fahnestock](https://github.com/gibfahn)
+29. [Vladimir de Turckheim](https://github.com/vdeturckheim)
+30. [Nicola Del Gobbo](https://github.com/NickNaso)
+31. [Shelley Vohr](https://github.com/codebytere)
+32. [Yihong Wang](https://github.com/yhwang)
+33. [Bryan English](https://github.com/bengl)
+34. [Benjamin Gruenbaum](https://github.com/benjamingr)
+35. [Waleed Ashraf](https://github.com/WaleedAshraf)
+36. [James M Snell](https://github.com/jasnell) - MAYBE
+37. [Gus Caplan](https://github.com/devsnek)- MAYBE
+38. [Matheus Marchini](https://github.com/mmarchini)- MAYBE
+39. [Tobias Nießen](https://github.com/tniessen)- MAYBE
+40. [Benjamin Zaslavsky](https://github.com/Tiriel)
+41. [Rachel White](https://github.com/rachelnicole)- MAYBE
+42. [Michael Dawson](https://github.com/mhdawson)- MAYBE
+43. [Richard Littauer](https://github.com/RichardLitt)- MAYBE
+44. [Joyee Cheung](https://github.com/joyeecheung)- MAYBE
+45. [Ujjwal Sharma](https://github.com/ryzokuken)
+46. [Franziska Hinkelmann](https://github.com/fhinkel)
+47. [Yosuke Furukawa](https://github.com/yosuke-furukawa)
+48. [Michaël Zasso](https://github.com/targos)
+
+
+### Attendees Remote
+There will be breakout sessions and we will do our best to optimize for remote attendance. More information will be posted on the agenda. Please let us know if you have any questions!
+
+1. [Bradley Meck](https://github.com/bmeck)
+2. [Liran Tal](https://github.com/lirantal)
+3. [Trivikram Kamat](https://github.com/trivikr)
+ 
+
+# Node Collaborator’s Dinner - 2018-05-31, 7pm
+Dinner will hosted by Node Foundation the first day of the Summit at Brauhaus Lemke, a short Uber ride from the AWS offices. 
+
+BRAUHAUS LEMKE
+AM HACKESCHEN MARKT
+DIRCKSENSTR., S-BAHNBOGEN 143
+10178 BERLIN-MITTE
+
+Please list if you have dietary restrictions below and submit a PR if you will be joining.
+
+### Guests
+1. [Myles Borins](https://github.com/MylesBorins) - no dietary restrictions
+2. [Matteo Collina](https://github.com/mcollina) - ?
+3. [Dan Shaw](https://github.com/dshaw) - ?
+4. [olivia](https://github.com/oe) - ?
+5. [Tracy Hinds](https://github.com/hackygolucky) 
+6. [Mark Hinkle](https://github.com/mrhinkle) - no dietary restrictions
+7. [Tierney Cyren](https://github.com/bnb) - vegetarian
+8. [Ujjwal Sharma](https://github.com/ryzokuken) - vegetarian
+9. [Franziska Hinkelmann](https://github.com/fhinkel)
+10. [Michaël Zasso](https://github.com/targos) - no dietary restrictions
+11. [Benjamin Zaslavsky](https://github.com/Tiriel) - allergies to shellfishes
+
```

Comment:
I will be attending, no restrictions

Reply:
Myself and @watson should be able to attend. I have no restrictions.

Reply:
@Qard @watson added yall to dinner rsvp

Reply:
I would also love to attend if still possible - no dietary restrictions

Reply:
I'll be able to attend as well. No dietary restrictions.